Yosemite/YosemiteTests/Stores/AccountStoreTests.swift

```diff
@@ -523,30 +476,6 @@ final class AccountStoreTests: XCTestCase {
         XCTAssertFalse(containsJCPSites)
     }
 
-    /// Verifies that `synchronizeSites` returns `true` when one site is JCP.
-    ///
-    func test_synchronizeSites_returns_true_when_one_site_is_jcp() throws {
```

Comment:
I believe I'm not fully understanding what's the behaviour we're looking for with this test:

As far as I understand it, despite we were passing the `isJetpackConnectionPackageSupported: false` flag and now we've removed it, when `synchronizeSites()` is called, it still should see there are sites where JCP is connected, however, `XCTUnwrap(result.get())` returns false and makes the test fail.

Should we be testing that `synchronizeSites()` should return false instead, if one or all of the sites are JCP?

Reply:
I just tried running this test case with `isJetpackConnectionPackageSupported: true` on `trunk`, and it looks like it's failing because two extra API requests are made for the JCP site that aren't mocked in `MockAccountRemote`. The test should pass after adding these two after the remote setup in the Given section:

```swift
remote.whenCheckingIfWooCommerceIsActive(siteID: 2, thenReturn: .success(true))
remote.whenFetchingWordPressSiteSettings(siteID: 2, thenReturn: .failure(NetworkError.notFound))
```

where `2` is the site ID for the JCP site.

Reply:
Oh! That's interesting! Thanks for the clear explanation. Fixed here: 65ad1089517a4e0567bc2c0ba0f3c8f6a05e973d